tasks/webfont.js

```diff
@@ -113,11 +114,13 @@ module.exports = function(grunt) {
 		// @todo Codepoint can be a Unicode code or character.
 		var currentCodepoint = o.startCodepoint;
 		if (!o.codepoints) o.codepoints = {};
+		if (o.codepointsFile) o.codepoints = codepointsFromFile();
 		o.glyphs.forEach(function(name) {
 			if (!o.codepoints[name]) {
 				o.codepoints[name] = getNextCodepoint();
 			}
 		});
+		saveCodepointsToFile();
```

Comment:
I thinks it’s better to write `if (o.codepointsFile) saveCodepointsToFile();` to be consistent with reading and to make it clear that we save codepoints only if `o.codepointsFile` is set.